src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -55,7 +55,7 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
     };
 
     socket.onerror = (event: Event) => {
-      console.log('onerror', event);
+      // console.log('onerror', event);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good that you've removed the `console.log` statement from production code. However, it might be beneficial to replace it with a proper error handling mechanism or logging system that can provide useful debugging information in a production environment when an error occurs.
```diff
- // console.log('onerror', event);
+ // Replace with your preferred logging or error handling method
+ logger.error(`Websocket error: ${event}`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is needed, will be added later